keadm/cmd/keadm/app/cmd/edge/reset_others.go

```diff
@@ -142,10 +124,29 @@ func NewOtherEdgeReset() *cobra.Command {
 
 // TearDownEdgeCore will bring down edge component,
 func TearDownEdgeCore() error {
-	ke := &util.KubeEdgeInstTool{Common: util.Common{}}
-	err := ke.TearDown()
+	extSystem, err := extsystem.GetExtSystem()
 	if err != nil {
-		return fmt.Errorf("TearDown failed, err:%v", err)
+		return fmt.Errorf("failed to get init system, err: %v", err)
+	}
+	service := util.KubeEdgeBinaryName
+	if extSystem.ServiceExists(service) {
+		klog.V(2).Info("edgecore service is exists")
+		if extSystem.ServiceIsActive(service) {
+			klog.V(2).Info("edgecore service is active, stopping it")
+			if err := extSystem.ServiceStop(service); err != nil {
```

Comment:
```suggestion
			if err = extSystem.ServiceStop(service); err != nil {
```
ditto, we have define `err` in line 127. Other parts can be modified together